### PR TITLE
Small Fixes

### PR DESCRIPTION
**Pull request by tcapelle**
This PR fixes the missing args for `show_images` and adds the `scheduler` to the learner object

### CODE REVIEW

File: miniai/sgd.py [hunk 1]
```diff
@@ -30,7 +30,9 @@
 # %% ../nbs/12_accel_sgd.ipynb 44
 class BaseSchedCB(Callback):
     def __init__(self, sched): self.sched = sched
-    def before_fit(self, learn): self.schedo = self.sched(learn.opt)
+    def before_fit(self, learn): 
+        self.schedo = self.sched(learn.opt)
+        learn.schedo = self.schedo
```
Comment (jph00) [inline miniai/sgd.py:35]: What's this for?

**Comment by jph00** [comment]
Thanks! Can you make this a more minimal diff without all the changes to images and other outputs, and remove the stacktrace that appears in one notebook?

File: miniai/sgd.py (hunk 1 above)
Comment (tcapelle) [reply miniai/sgd.py:35]: To have access to the scheduler inside the Learner object. This way, I can monitor the learning rate or other scheduler params.

**Comment by tcapelle** [comment]
Ok, I did `nbdev_clean --clear_all`

**Comment by jph00** [comment]
Unfortunately that deleted all the outputs!

**Comment by tcapelle** [comment]
:/

**Comment by tcapelle** [comment]
Ok, re run everything, but lot's of warning on M1Pro

**Comment by tcapelle** [comment]
I will close this then... don't know how to resolve the conflicts

File: miniai/datasets.py [hunk 2]
```diff
@@ -61,7 +61,7 @@ def subplots(
     if figsize is None: figsize=(ncols*imsize, nrows*imsize)
     fig,ax = plt.subplots(nrows, ncols, figsize=figsize, **kwargs)
     if suptitle is not None: fig.suptitle(suptitle)
-    if nrows*ncols==1: ax = array([ax])
+    if nrows*ncols==1: ax = np.array([ax])
```
Comment (jph00) [inline miniai/datasets.py:64]: This change seems to already be in master

File: miniai/sgd.py (hunk 1 above)
Comment (jph00) [reply miniai/sgd.py:35]: Can you do that using the optimiser hyperparams instead?

**Comment by jph00** [comment]
I'll fix `show_images` directly.

File: miniai/sgd.py (hunk 1 above)
Comment (tcapelle) [reply miniai/sgd.py:35]: oh, you are right =)

Comment (tcapelle) [reply miniai/sgd.py:35]: `opt.param_groups[0]["lr"]` returns the current learning rate from the `optimizer` object

File: miniai/datasets.py (hunk 2 above)
Comment (tcapelle) [reply miniai/datasets.py:64]: =)
